socorro/unittest/external/es/test_correlations.py

```diff
@@ -53,3 +124,90 @@ def test_create_correlations_index(self):
 
         # We should be able to create that index again without any errors.
         correlations.create_correlations_index(es_index)
+
+
+class TestCoreCounts(BaseTestCorrelations):
+
+    def test_store(self):
+        correlations = CoreCounts(config=self.config)
+        today = datetime.datetime.utcnow()
+        correlations.store(
+            sample_core_counts_summary,
+            today.strftime('%Y%m%d'),
+            'core-counts',
+            'Firefox_43.0.1'
+        )
+        es_index = correlations.get_index_for_date(today.date())
+        correlations.create_correlations_index(es_index)
+        self.indices_for_deletion.append(es_index)
```

Comment:
I get the idea but what do you suggest? The defaults have the word "test" in them.